package.json

```
@@ -2,6 +2,10 @@
   "name": "island.is",
   "version": "0.0.1",
   "license": "MIT",
+  "engines": {
+    "node": ">=12",
+    "yarn": ">=1.20"
```

Should we maybe pin down a major version here?

```suggestion
    "node": "^12.20",
    "yarn": "^1.20"
```

---

@eirikurn has been considering pinning the repo to node@^14 at some point in the near future — so this sort of thing should just wait until then.